docs/modules/clusters/pages/ucn-enable.adoc

```diff
@@ -0,0 +1,70 @@
+= Enable {ucn}
+:description: For security reasons, you must enable {ucn} using static configuration. You can do this programmatically, or using the Hazelcast configuration file.
+:page-enterprise: true
+:page-beta: false
+
+{description}
+
+NOTE: If using dynamic persistence, this must be enabled in the Hazelcast configuration file. For further information persisting changes, see the xref:configuration:dynamic-config.adoc#manual[Dynamic Configuration] Overview topic under the Configure and Manage Clusters section.
+
+== Enable Programmatically
+
+To enable {ucn} programmatically, create a _member.java_ file, add the following, then save the file to your configuration folder:
```

Comment:
nit: if someone was to actually try this, they would have a compiler error due to mismatch of filename & class name.

```suggestion
To enable {ucn} programmatically, create a _Member.java_ file, add the following, then save the file to your configuration folder:
```